pkg/alertmanager/api_test.go

```diff
@@ -151,6 +155,51 @@ template_files:
 	}
 }
 
+func TestMultitenantAlertmanager_DeleteUserConfig(t *testing.T) {
+	storage := chunk.NewMockStorage()
+	alertStore := objectclient.NewAlertStore(storage, log.NewNopLogger())
```

Comment:
I'm going to deprecate the storage based on the chunk objstore client soon. Would you mind switching to the new bucket-client one?

```suggestion
	alertStore := prepareInMemoryAlertStore()
```